Containers/Deploy-.NET-Core-app-in-Windows-Containers-to-ECS/Readme.md

```diff
@@ -235,6 +250,17 @@ The container's public IP address is then listed in the *Network bindings* secti
 
 Click on the URL under the *External Link* section, and the ASP.NET Core site will load in your browser.
 
-![App](media/image5.png)
+![App](media/image5_updated.PNG)
```

Comment:
Can we keep the same filename, rather than append _updated? And keep the file extension lower case.

Reply:
Done